onnx/cpp2py_export.cc

```diff
@@ -91,21 +91,32 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
       .value("COMMON", OpSchema::SupportType::COMMON)
       .value("EXPERIMENTAL", OpSchema::SupportType::EXPERIMENTAL);
 
-  defs.def("has_schema", [](const std::string& op_type) -> bool {
-    return OpSchemaRegistry::Schema(op_type) != nullptr;
+  defs.def("has_schema", [](const std::string& op_type,
+			    const std::string& domain) -> bool {
+    return OpSchemaRegistry::Schema(op_type, domain) != nullptr;
+  }).def("has_schema", [](const std::string& op_type) -> bool {
```

Comment:
pybind11 supports specifying default value for function argument, so no need to define twice for different function signatures.

Reply:
I tried default value but pytest gave me errors 
```   
def test_typecheck(self):
>       defs.get_schema("Conv")
E       TypeError: get_schema(): incompatible function arguments. The following argument types are supported:
E           1. (arg0: str, arg1: int, arg2: str) -> onnx.onnx_cpp2py_export.defs.OpSchema
E
E       Invoked with: 'Conv'

onnx\test\schema_test.py:12: TypeError
```

Reply:
Have you annotated the default arguments like this: http://pybind11.readthedocs.io/en/stable/basics.html#default-arguments ?

Reply:
yes... I also tried none(true), not working as well... also I feel a little bit uncertain want should the default value be, because we already have a default one defined in schema.h

Reply:
the default value should match the one in c++
btw the error you got seems to be from get_schema, not has_schema

Reply:
they have the same error so I copied one of it.

Reply:
ok as this is just implementation details that doesn't affect user interface, let's get this in first and we can revisit it later.